src/ListComponent.js

```diff
@@ -8,9 +8,12 @@ import InputGroup from "react-bootstrap/InputGroup";
 import FormControl from "react-bootstrap/FormControl"; 
 import ListGroup from "react-bootstrap/ListGroup"; 
 
+
 const ListComponent = (props) => { 
     const [input, setInput] = useState('');
     const [list, setList] = useState([]);
+    const [done, complete] = useState(true);
```

Comment:
I am new to react/js so take this with a grain of salt...

I think this should be inside the taskToAdd object, this way if a task is complete it is stored with all of that task's values/variables. See next comment for what I mean sorta.

Reply:
Well apparently GitHub does not allow you to add a comment to an unchanged line...annoying or am I using it wrong?

I meant to add like `done: false,` to new line #22